src/test/java/de/aktey/scanndal/classmappertest/ClassMapperTestClass.java

```diff
@@ -10,4 +13,10 @@
     public String foo() {
         return "bar";
     }
+    public void forEachLoop() {
+        List<String> list = Arrays.asList("a", "b");
+        list.forEach( (l) -> {
+            System.out.println(l);
```

Comment:
Hi Ruben, these lines are meant to test that the class scanner is able to read classes with Java 1.8 feature. E.g. for the dynamic creation of the synthetic lambda type and the
lambda object the invokedynamic byte code is used (which was actually introduced in v7).
Move the test to a separate test class?